### ISSUE TITLE
Support new special function calling convention (ClojureScript 2665+ compat)

### ISSUE THREAD

**Issue by cemerick** [cemerick/piggieback#38]

**Comment by swannodette** [comment]
```cljs
(fn self
   ([repl-env env form]
      (self repl-env env form nil))
   ([repl-env env form opts]
      ...))
```

**Comment by cemerick** [comment]
This fix is now available in `0.1.5-SNAPSHOT`, for those that care. I'm going to use it for a bit with the recently-updated austin, etc., before releasing to make sure I'm not missing anything.
